staging/src/k8s.io/legacy-cloud-providers/gce/support.go

```diff
@@ -66,6 +66,9 @@ func (l *gceRateLimiter) Accept(ctx context.Context, key *cloud.RateLimitKey) er
 	return nil
 }
 
+// Observe is a no-op func to satisfy cloud.RateLimiter
```

Comment:
where is defined this interface? https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/cloud.go

Reply:
oh, is here https://github.com/GoogleCloudPlatform/k8s-cloud-provider/pull/77